owncloudApp/src/androidTest/java/com/owncloud/android/settings/security/OCSettingsSecurityTest.kt

```diff
@@ -143,7 +143,7 @@ class OCSettingsSecurityTest {
         intending(hasAction(PatternLockActivity.ACTION_REQUEST_WITH_RESULT)).respondWith(intentResult)
         onView(withText(R.string.prefs_pattern)).perform(click())
         assertTrue(mPrefPattern.isChecked)
-        onView(withText(R.string.pattern_stored)).check(matches(isDisplayed()))
+        //onView(withText(R.string.pattern_stored)).check(matches(isDisplayed()))
```

Comment:
Same here

Reply:
The point here is that we could delete the whole file. The security section is now tested in a different way. But if we want to remove the old implementation in the last step, I agree with you that we should remove the line.

Reply:
Let's keep the class until the last PR and when we have all the preferences updated to AndroidX, we can delete these unused files. I'll remove the out-commented lines until then, thx!